ext/cl_khr_external_semaphore.asciidoc

```diff
@@ -317,6 +327,36 @@ imported payload, the semaphore's prior permanent payload will be restored.
 Please refer to handle specific specifications for more details on transference and
 permanence requirements specific to handle type.
 
+To import a handle of type {CL_SEMAPHORE_HANDLE_SYNC_FD_KHR} into an existing semaphore, call the function
```

Comment:
Discussed in the August 29th teleconference.  I think either:

1. This sentence is stale and should be updated to include all handle types (it sounded like this was the intent).
2. Or, we need to define an error code when this API is used with a non-SYNC_FD handle type.